src/amplitude/client.py

```diff
@@ -43,6 +45,11 @@ def __init__(self, api_key: str, configuration: Config = Config()):
             configuration (amplitude.config.Config, optional): The configuration of client instance. A new instance
                 with default config value will be used by default.
         """
+        logging.basicConfig(filename="amplitude.log",
+                            filemode='a+',
+                            format='%(asctime)s - %(code)s - %(message)s - %(event)s',
+                            level=logging.INFO
+                            )
```

Comment:
I think it's better to let customer config logging. We just need to put log messages in the right log level.

Reply:
But if we only set log level and the number of events is huge, logs in terminal is overwhelming. Besides, if users want to monitor the process and use callback function, it will be mixed with logs and be hard to distinguish.

Reply:
In production, the log level usually set to ERROR, logs like INFO and DEBUG won't output. If we config logging to file. It may be conflict with user's own logging config and do disk I/O, use disk space.

Reply:
I deleted this block of code, so the logging level is changed back to debug again.